### PR TITLE
improved error messages and added verification scripts

### PR DESCRIPTION
**Pull request by pabloruiz55**

### CODE REVIEW

**Comment by Everhusk** [comment]
Let's move this into the polymath-allocations repo.

**Comment by pabloruiz55** [comment]
Ok, let's move it once we freeze the code here. Should be final now. We might want to test it live one more time today/tomorrow.

File: scripts/data/presale.csv [hunk 1]
```diff
@@ -0,0 +1,13 @@
+0xDE5902674ee6a5A5746593Ef1e41022C74aBaB91,26880000
+0x7D843266918804836073bF44ad6193E677961D3B,4000000
+0x2b90bbfeC6d4069236346f7604aBE0bF20B3bf9D,4000000
+0xf89d8021ea119402c75087493d1477f22ef9bb19,100000
+0x4b411e2d18d256ba8b1d94382f2d2ac72dc75378,10000
+0x5a6fe61709eb43b8c1a689eff5b2680a394c6466, 100
+0x93597de43e05130862649f2c008e94e238a7d9a7, 200
+0xf2096409fdc38ac4a388890d9532752fc96af9d9, 333
+0x2f1c806f2a2c10349e4c03c04a524c20b39204b5, 444
+0x79e2e4092254d1b2fbf1ce1ae36f41f0ea402e00, 56700000000000000
```
Comment (jay) [inline scripts/data/presale.csv]: I'm curious why is this 56 quadrillion?

Comment (pabloruiz55) [reply scripts/data/presale.csv]: Just a test to make sure we are catching an error with allocations higher than the supply.

**Comment by Everhusk** [comment]
We're going to have to pull this out in a bit, but will get you the +1 PR points

**Comment by pabloruiz55** [comment]
Thanks, after all my name is PR.

**Pull request merged**